test/functional/editor_controller_test.rb

```diff
@@ -69,7 +77,7 @@ def setup
 
   test "should show title form input if title parameter present" do
```

Comment:
I'd like to mark this as "legacy" as well, and also can we please make copies of each test that is now of the "legacy" form, and adapt them to test the new form template? 

Then we'll be done!